apps/src/lib/node/ledger/shell/prepare_proposal.rs

```diff
@@ -179,12 +179,12 @@ where
     }
 
     /// Builds a batch of DKG decrypted transactions
-    // TODO: we won't have frontrunning protection until V2 of the Anoma
+    // TODO: we won't have frontrunning protection until V2 of the Namada
     // protocol; Namada runs V1, therefore this method is
     // essentially a NOOP, and ought to be removed
     //
     // sources:
-    // - https://specs.anoma.net/main/releases/v2.html
+    // - https://specs.namada.net/main/releases/v2.html
```

Comment:
leaving this as a note: we should revert this change, as it was correct before to mention Anoma here